torch/csrc/distributed/rpc/python_functions.cpp

```diff
@@ -128,8 +153,14 @@ std::shared_ptr<FutureMessage> pyRpcBuiltin(
   py::gil_scoped_release release;
   auto scriptCall = std::make_unique<ScriptCall>(op, std::move(stack));
   auto agent = RpcAgent::getCurrentRpcAgent();
-  return sendMessageWithAutograd(
+  auto responseMessageFuture = sendMessageWithAutograd(
       *agent, dst, std::move(*scriptCall).toMessage(), false, rf);
+
+  // Notice, even we can get the JIT type of the Python object from the op
+  // schema, there is no need to do that, because the return value will be
+  // passed back to Python land eventually.
+  return std::make_shared<torch::jit::PythonFutureWrapper>(
+      wrapFutureMessageInJitFuture(responseMessageFuture));
```

Comment:
same, we should return a ivalue::future here.

Reply:
I think it's not strictly necessary to align with `torchscript_functions.cpp`. It has the constraint that we don't wan it to have Python dependency, but for "python_funcitons.cpp", it has alreay used a lot of pybind utilities, so it doesn't matter.

The purpose of my moving it to here is that I want to put the call, `pythonRpcHandler.handleException(value);` to be as close as the main logic, so that we don't forget this special case. Imagine we leave this single line in `rpc/init.cpp`.

Reply:
We moved deserialize Messsge into py::object from user thread to RPC server thread.